ebu_tt_live/bindings/converters/ebutt3_ebuttd.py

```diff
@@ -155,6 +156,11 @@ def convert_head(self, head_in, dataset):
             else:
                 new_elem.append(item)
 
+        metadata = headMetadata_type()
+        metadata.documentMetadata = documentMetadata()
+        metadata.documentMetadata.conformsToStandard = ['http://www.w3.org/ns/ttml/profile/imsc1/text', 'urn:ebu:tt:distribution:2018-04']
```

Comment:
You might be able to do this and the previous line all in one go, like:

```
metadata.documentMetadata = documentMetadata(conformsToStandard = ['x','y'])
```

Might be worth trying, for a minor gain.